ops/docker-compose.yml

```diff
@@ -62,6 +62,7 @@ services:
         # connect to the 2 layers
         DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT: http://l1_chain:8545
         DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT: http://l2geth:8545
+        DATA_TRANSPORT_LAYER__SYNC_FROM_L2: 'true'
```

Comment:
Why is this needed? I thought the DTL was able to be consumed by both replicas and verifiers?

Reply:
without this switch the dtl wasn't syncing with L2 automatically